config.json

```diff
@@ -1402,6 +1402,19 @@
           "unsafe"
         ]
       },
+      {
+        "slug": "resistor-color",
+        "name": "Resistor Color",
+        "uuid": "51c31e6a-b7ec-469d-8a28-dd821fd857d2",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4,
+        "topics": [
+          "enums",
+          "external_crates",
+          "format"
+        ]
+      },
```

Comment:
I see you've placed this last among the non-deprecated exercises. It's not a bad guess, but it should actually go somewhere among the difficulty-4 problems. For practice exercises, the ordering of the exercises in this list determines the "next recommended exercise" flow, so it's best to keep the ordering sensible.